script/Arcueid/Day 5/05_00_ARC05_9_2.txt

```diff
@@ -168,20 +171,20 @@ And over there―――
 [sha:e1201e09bcd131d9b607ac385ad6408a56ff237c]{
 -- Page 654, Offset 10414.
 -- 説明する余裕はない。
-There's no time to explain.
+There's no more time to explain.
```

Comment:
mild pref for previous version

Reply:
The way they get interrupted by Vlov makes me feel like "no more" fits more in the scene.